docs/quickstarts/csharp-quickstart.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: docs
+title: C# Quickstart
+---
+
+<h1 class="page-header">C# Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in C# with a simple working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+* Windows: .NET Framework 4.5+, Visual Studio 2013 or 2015. You may also need the Nuget executable 
+* Linux: Mono 4+, MonoDevelop 5.9+ (with NuGet add-in installed)
+* Mac OS X: Xamarin Studio 5.9+
+
+### How to use Grpc
```

Comment:
"How to use gRPC" sounds a bit like it's telling you how to actually write gRPC code, how about "Adding gRPC to a project"